From 7d0b9aba38eba0d4821302ee8ac1e81f220091f9 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robux4@videolabs.io>
Date: Wed, 25 Jan 2017 16:22:18 +0100
Subject: [PATCH 18/55] Revert "direct3d11: avoid crashing later in the
 decoder"

This reverts commit 57ff42122f34fee45d78937467c0a5285cabb911.

We are going to need to allow more textures with software decoding mixed with
direct rendering.
---
 modules/video_output/win32/direct3d11.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/modules/video_output/win32/direct3d11.c b/modules/video_output/win32/direct3d11.c
index 472f2b6..3305155 100644
--- a/modules/video_output/win32/direct3d11.c
+++ b/modules/video_output/win32/direct3d11.c
@@ -492,11 +492,6 @@ static picture_pool_t *Pool(vout_display_t *vd, unsigned pool_size)
     if ( vd->sys->pool != NULL )
         return vd->sys->pool;
 
-    if (pool_size > 30) {
-        msg_Err(vd, "Avoid crashing when using ID3D11VideoDecoderOutputView with too many slices (%d)", pool_size);
-        return NULL;
-    }
-
 #ifdef HAVE_ID3D11VIDEODECODER
     picture_t**       pictures = NULL;
     unsigned          picture_count = 0;
-- 
2.10.1.windows.1

